worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/PlatformManager.java

```diff
@@ -284,6 +291,21 @@ public PlatformCommandManager getPlatformCommandManager() {
         return platformCommandManager;
     }
 
+    /**
+     * Get the executor service. Internal, not for API use.
+     *
+     * @return the executor service
+     */
+    public ListeningExecutorService getExecutorService() {
```

Comment:
Potentially the platform manager should expose the `Lifecycled` and the `valueOrThrow` can be moved up to the `WorldEdit` class. That would allow things to add listeners to the `Lifecycled` events if it becomes needed at some point. But also, this isn't API so we can change it at any time if that's needed.